content/help/ops/setup/app-health-check/index.md

```diff
@@ -171,6 +171,25 @@ NAME                             READY     STATUS    RESTARTS   AGE
 liveness-http-975595bb6-5b2z7c   2/2       Running   0           1m
 {{< /text >}}
 
+#### Probe rewrite cleanup
+
+Remove the service and redeploy Istio with Helm option `sidecarInjectorWebhook.rewriteAppHTTPProbe=false`.
```

Comment:
This intro cover the what readers will do on this section but they still need to know why and when they need to do it.